stage #4/module #5. AWS/theory/aws_task.md

```diff
@@ -0,0 +1,51 @@
+# AWS
```

Comment:
@a1exanddrovich Could you please add links to the theories (dynamodb, lambda, vpc) on the main page.

The possible structure can be:

# Theoretical part

link to vpc
link to dynamodb
link to lambda

# Practice part

here place conditions how to choose which type of practical part students should select. if nothing can be completed due to any circumstances, they should go to quiz part

## Practice in AWS Console

## Practice with LocalStack

# Quiz

# Demo

# Additional materials

(copied links with additional materials here)

Reply:
added